crates/wasmtime/src/runtime/vm/traphandlers.rs

```diff
@@ -433,7 +446,7 @@ where
 
     let result = CallThreadState::new(store.0, old_state).with(|cx| match store.0.executor() {
         // In interpreted mode directly invoke the host closure since we won't
-        // be using host-based `setjmp`/`longjmp` as that's not going to save
+        // be using host-based `etjmp`/`longjmp` as that's not going to save
```

Comment:
👀

Reply:
I believe [this is an ET jmp](https://i.imgflip.com/8bd001.jpg), for reference